shared/src/main/scala/squants/Dimension.scala

```diff
@@ -72,7 +96,11 @@ trait Dimension[A <: Quantity[A]] {
       case _                             ⇒ Failure(QuantityParseException(s"Unable to parse $name", s))
     }
   }
-  private lazy val QuantityString = ("^([-+]?[0-9]*\\.?[0-9]+(?:[eE][-+]?[0-9]+)?) *(" + units.map { u: UnitOfMeasure[A] ⇒ u.symbol }.reduceLeft(_ + "|" + _) + ")$").r
+
+  private lazy val regexUnits = units
```

Comment:
Oops, that was left behind from figuring out how to handle unicode, I'll switch it back to how it was before.